test/rqg/mzcompose.py

```diff
@@ -127,19 +134,36 @@ class Workload:
         reference_implementation=None,
         validator="QueryProperties,RepeatableRead",
     ),
+    # Added as part of MaterializeInc/materialize#25340.
+    Workload(
+        name="left-join-stacks",
```

Comment:
Please add a new rqg test in ci/nightly/pipeline.template.yml using `left-join-stacks`, then we will run it in Nightly automatically too.

Reply:
Will do. I'll probably use shorter `duration` and `queries` for this workload, though (I think the current grammar doesn't open a very big query space, so running `1000` queries should be enough to catch regressions (the current one is caught with 10 queries for example).